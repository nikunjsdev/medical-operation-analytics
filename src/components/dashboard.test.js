import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from './dashboard';

const setup = () => {
    const component = shallow(<Dashboard />)
    return {
        component
    }
}

describe('Dashboard Component', () => {
    it('should render Dashboard component', () => {
        const { component } = setup();
        expect(component).toMatchSnapshot();
    });
})
