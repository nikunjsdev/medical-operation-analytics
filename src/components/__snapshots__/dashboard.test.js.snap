// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Component should render Dashboard component 1`] = `
<div
  className="App"
>
  <WithStyles(ForwardRef(AppBar))
    position="static"
  >
    <WithStyles(ForwardRef(Toolbar))>
      <WithStyles(ForwardRef(Typography))
        variant="h6"
      >
        Medical Operation Analytics
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
  <WithStyles(ForwardRef(Grid))
    alignItems="center"
    container={true}
    direction="column"
    justify="center"
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      key="temperature"
      xs={true}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <b>
          Temperature
        </b>
        <span
          className="makeStyles-text-3"
        >
          0
        </span>
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      key="pressure"
      xs={true}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <b>
          Air Pressure
        </b>
        <span
          className="makeStyles-text-3"
        >
          0
        </span>
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      key="humidity"
      xs={true}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <b>
          Humidity
        </b>
        <span
          className="makeStyles-text-3"
        >
          0
        </span>
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</div>
`;
